keps/sig-node/3085-pod-conditions-for-starting-completition-of-sandbox-creation/README.md

```diff
@@ -1069,14 +1100,17 @@ in back-to-back releases.
 #### Alpha
 
 - Kubelet will report pod sandbox conditions if the feature flag `PodReadyToStartContainersCondition` is enabled.
-- Initial e2e tests completed and enabled.
+- E2E tests added for pod conditions.
+- E2E test for sandbox condition if pod fails to mount volume.
 
 #### Beta
 
+- Condition is moved from a package constant in Kubelet to a [API Defined Condition](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L2834)
 - Gather feedback from cluster operators and developers of services or controllers that consume these conditions.
 - Implement suggestions from feedback as feasible.
 - Feature Flag defaults to enabled.
-- Add more test cases and link to this KEP.
+- Add test case for graceful shutdown.
+- Add test case for sandbox condition if pod fails to mount volume from a missing secret.
```

Comment:
Some comments about PRR:

> Does enabling the feature change any default behavior?

I somewhat disagree that it's no default change - we actually start reporting a new condition, so the pod object will look differently without any action from the user.

> Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?

Can you please describe the test in more details?
https://github.com/kubernetes/enhancements/pull/3658/files is a nice example

Reply:
I didn't do that test or update that question (as it was there since alpha) but I'll run a test case today and verify this.

Reply:
I'm having some trouble finding a way to actually test this.  What suggestions do you have for rollout/rollbacks?  If need be, I can add an openshift release test for this (and test the upgrade/downgrade).  I don't think I'll be able to do that in this week though.  

Can I add this as a required code change for beta?

I tried to see if I could test this with Kind but I am not sure how to set feature-gates on kubelet and restart kubelet.

Reply:
I updated this with my test case of deploying a kind cluster with feature on/off/on.  I hope that is sufficient.  

if not, I will continue to find a way to test this.

Reply:
Thanks - that looks reasonable to me.